libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -195,6 +195,102 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     AP_SUBGROUPINFO(robotis, "_ROB_",  22, SRV_Channels, AP_RobotisServo),
 #endif // HAL_BUILD_AP_PERIPH
 
+#if (NUM_SERVO_CHANNELS >= 17)
```

Comment:
I think we should have an enable param to get more than 16
Maybe SERVO32_ENABLE ?

Reply:
Possibly.. because this creates a ton of params

Reply:
Added `SERVO32_ENABLE` but not enabled it anywhere yet